src/libraries/tests.proj

```diff
@@ -264,7 +264,6 @@
                       Condition="'$(TestPackages)' == 'true'" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)*\tests\**\*.TrimmingTests.proj"
                       Exclude="@(ProjectExclusions)"
-                      BuildInParallel="false"
```

Comment:
When I originally added this, I also added a `ContinueOnError` property. Can that be removed now?

https://github.com/dotnet/runtime/commit/254ef0f7f7f429ec238735fe6132805e3c38a19f#diff-5ef2fd16f58de91bff2f05ad601ab6ff000c0febabb2312d232f20cfa132a71bR14

Reply:
Looks like test projects already have a ErrorAndContinue so yeah we might want to remove the ContinueOnError specific for Trimming and just behave the same as other regular test projects.